stock_measuring_device/models/measuring_device.py

```diff
@@ -0,0 +1,112 @@
+# Copyright 2021 Camptocamp SA
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html)
+import logging
+
+from odoo import _, fields, models
```

Comment:
```suggestion
from odoo import fields, models
```

And change all occurrences with `self.env._`
This provides a faster gettext execution due to it not being to inspect the stack to identify the target language